orca/templates/db/resources/usr/local/bin/_liferay_common.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+function block_begin {
+	text="${1}"
```

Comment:
In these cases we don't need to define the text, just use the ${1} directly.

Reply:
This is a kind of proposal of making the code more readable (from my perspective).
What's your thought?